.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,131 @@
+name: Benchmark
+
+on:
+  pull_request:
+    branches: [master]
+    types: [labeled, synchronize]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    if: >-
+      (github.event.action == 'labeled' && github.event.label.name == 'performance') ||
+      (github.event.action == 'synchronize' && contains(github.event.pull_request.labels.*.name, 'performance'))
+
+    env:
+      MSG_FOOTER: |-
+        <br/>
+
+        Workflow: [${{ github.run_id }}](/${{ github.repository }}/actions/runs/${{ github.run_id }})
+        *Adding new commits will generate a new report*
+
+    steps:
+      - name: Post comment
+        uses: jungwinter/comment@v1
+        id: create_comment
+        with:
+          type: create
+          body: |
+            Started a benchmark for this pull request.
+            This comment will be updated with the results.
+            ${{ env.MSG_FOOTER }}
+          issue_number: ${{ github.event.number }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: actions/checkout@v2
+        with:
+          repository: yewstack/js-framework-benchmark
+
+      - name: Configure benchmark
+        run: |
+          search="{ git = \"https://github.com/jstarry/yew\", branch = \"keyed-list\" }"
+          replace="{ git = \"${{ github.event.pull_request.head.html_url }}\", branch = \"${{ github.event.pull_request.head.ref }}\" }"
+          input=$(cat frameworks/keyed/yew/Cargo.toml)
+          output=${input//"$search"/"$replace"}
+          if [[ "$input" == "$output" ]]; then
+            echo "ERROR: failed to configure Cargo.toml"
+            exit 1
+          fi
+          echo $output > frameworks/keyed/yew/Cargo.toml
```

Comment:
This is very hacky but it does the trick. If we want to make this cleaner I think we should tackle it over at `yewstack/js-framework-benchmark`.